src/current/_includes/v24.1/essential-metrics.md

```diff
@@ -50,6 +50,21 @@ The **Usage** column explains why each metric is important to visualize in a cus
 
+{% if include.deployment == 'self-hosted' %}
+## Network
+
+| <div style="width:225px">CockroachDB Metric Name</div> | <div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)</div> |<div style="width:150px">Description</div>| Usage |
+| ------------------------------------------------------ | --------------------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| rpc.connection.avg_round_trip_latency                  | rpc.connection.avg_round_trip_latency                     | Sum of exponentially weighted moving average of round-trip latencies, as measured through a gRPC RPC. Dividing this Gauge by rpc.connection.healthy gives an approximation of average latency, but the top-level round-trip-latency histogram is more useful. Instead, users should consult the label families of this metric if they are available (which requires prometheus and the cluster setting `server.child_metrics.enabled`); these provide per-peer moving averages. This metric does not track failed connection. A failed connection's contribution is reset to zero. | This metric is helpful in understanding general network issues outside of CockroachDB that could be impacting the user’s workload. |
+| rpc.connection.failures                                | rpc.connection.failures.count                             | Counter of failed connections. This includes both the event in which a healthy connection terminates as well as unsuccessful reconnection attempts. Connections that are terminated as part of local node shutdown are excluded. Decommissioned peers are excluded. | See Description. |
+| rpc.connection.healthy                                 | rpc.connection.healthy                                    | Gauge of current connections in a healthy state (i.e. bidirectionally connected and heartbeating) | See Description. |
+| rpc.connection.healthy_nanos                           | rpc.connection.healthy_nanos                              | Gauge of nanoseconds of healthy connection time. On the prometheus endpoint scraped with the cluster setting `server.child_metrics.enabled` set, the constituent parts of this metric are available on a per-peer basis and one can read off for how long a given peer has been connected. | See Description. |
```

Comment:
```suggestion
| rpc.connection.healthy_nanos                           | rpc.connection.healthy_nanos                              | Gauge of nanoseconds of healthy connection time. On the Prometheus endpoint scraped with the cluster setting `server.child_metrics.enabled` set, the constituent parts of this metric are available on a per-peer basis and one can read off for how long a given peer has been connected. | See Description. |
```

Reply:
I don't quite understand what "one can read off for how long..." means? There's an instance in the last metric in this table as well.

Reply:
updated descriptions for `rpc.connection.healthy_nanos` and `rpc.connection.unhealthy_nanos` for clarity.